roles/cloud_connector/defaults/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+foreman_cloud_connector_url: https://foreman.example.com
+foreman_cloud_connector_username: admin
+foreman_cloud_connector_password: changeme
+foreman_cloud_connector_console_redhat_host: cert.cloud.redhat.com
+foreman_cloud_connector_config_file: /etc/rhc/workers/rhc-worker-cloud-connector.toml
```

Comment:
Since the directive name is [derived](https://github.com/RedHatInsights/yggdrasil/blob/b35da0f6e66c6fbac58ec9839925bafdddf28b0b/cmd/yggd/worker.go#L42) from configuration file name, we have to call this file `foreman_rh_cloud.toml`.
I know it's inconvenient, and I don't like it either.